airflow/providers/google/cloud/example_dags/example_azure_fileshare_to_gcs.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from datetime import timedelta, datetime
+
+from airflow import DAG
+from airflow.providers.google.cloud.transfers.azure_fileshare_to_gcs import AzureFileShareToGCSOperator
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': ['airflow@example.com'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 1,
+    'retry_delay': timedelta(minutes=5),
+}
+
+with DAG(
+    dag_id='aci_example',
+    default_args=default_args,
+    schedule_interval=timedelta(1),
+    start_date=datetime(2018, 11, 1),
+    tags=['example'],
+) as dag:
+    # [START howto_operator_azure_fileshare_to_gcs_basic]
+    t1 = AzureFileShareToGCSOperator(
+        task_id='sync_azure_files_with_gcs',
+        share_name='my-azure-share',
+        directory_name='mydir/anotherdir',
+        prefix='data/customers-201804',
+        wasb_conn_id='azure_fileshare_default',
+        gcp_conn_id='google_cloud_default',
+        dest_gcs='gs://my.gcs.bucket/some/customers/',
+        replace=False,
+        gzip=True,
+        google_impersonation_chain=['account1', 'account2', 'account3'],
```

Comment:
I'm afraid that this won't work, @olchas

Reply:
Yes, the correct values for `google_impersonation_chain` are either a single service account email or a list of emails. So, I would either keep the `google_impersonation_chain` as `None`, or let the user specify it via environment variable (with `None` as the default value).

Reply:
ok